js/data.js

```diff
@@ -1 +1,24 @@
 /* exported data */
+var data = {
+  view: 'images',
+  comments: [{
+    imageId: '',
+    textValue: ''
```

Comment:
I wouldn't start these with an empty value, I imagine that will cause problems. You should have both `comments` and `favorites` start with an empty array. If you want to keep these to demonstrate the shape of the data they will store, you can leave it as a code comment within the array as a reference 🤓